src/test/ui/parser/removed-syntax-field-let-2.stderr

```diff
@@ -0,0 +1,33 @@
+error: expected identifier, found keyword `let`
+  --> $DIR/removed-syntax-field-let-2.rs:2:5
+   |
+LL |     let x: i32,
```

Comment:
I'm actually not a fan of this removal span that overlaps with the `let` keyword span -- we don't usually care that spacing gets messed up by suggestions, which is why these two spans differ.

I think we should just adjust the "remove this `let` keyword" span to remove just the keyword, and the user can run `rustfmt` to fix the leftover spaces :stuck_out_tongue_closed_eyes: thoughts?

Reply:
Suggestions are not aware of global formatting rules, nor are they typically designed to handle cases where spacing, newlines, etc. are messed up because of code suggestions.  At least, as long as the suggestion _renders_ in a way that the user can understand, I feel like this is not the responsibility of the parser's error messages.

Reply:
Given that `rustfmt` absolutely refuses to format a `struct` that has `let ident: ty` inside of it, I think formatting is a step that will need to be done regardless :sweat_smile:

Reply:
I don't have a strong preference here - I'd probably bias towards keeping the four-character span. Feel free to change it if you want.